scripts/ci/ie_setup.bat

```diff
@@ -45,3 +45,13 @@ reg add "HKEY_CURRENT_USER\Software\Microsoft\Internet Explorer\Main" ^
 :: Disable cross-origin XHRs. Sauce enables this in IE by default.
 @ECHO OFF
 REG ADD "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\Zones\3" /v 1406 /t REG_DWORD /d 3 /f
+
+@echo off
+setlocal
+
+set "URL=https://raw.githubusercontent.com/shicks/closure-library-temp/master/CloseAdobeDialog.exe"
```

Comment:
Hopefully it does! Thanks for working on this. It really hurts to see so many tests fail because of these.